avm/res/sql/instance-pool/tests/e2e/waf-aligned/main.test.bicep

```
@@ -0,0 +1,63 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
```

Incorrect description. Should be something like

```suggestion
metadata name = 'WAF-aligned'
metadata description = 'This instance deploys the module in alignment with the best-practices of the Well-Architected Framework.'
```